setup.cfg

```diff
@@ -0,0 +1,66 @@
+[metadata]
+name = lsst_versions
+description = Calculate package versions following LSST code conventions.
+author = Rubin Observatory Data Management
+url = https://github.com/lsst/lsst_versions
+classifiers =
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+version = attr: lsst_versions.__version__
+readme = file: README.md
+
+[options]
+# zip_safe can't be used for mypy compatibility
+zip_safe = False
+package_dir=
+    =python
+packages=find:
+setup_requires =
+  setuptools >=46.0
+install_requires =
+  setuptools >= 46.0
+  tomli>1.0.0
+  GitPython >= 3.1
+  packaging
+tests_require =
```

Comment:
FYI tests_require is deprecated (https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#optional-dependencies).

What we do in SQuaRE is set create a "dev" extra for all of our testing and documentation dependendencies. i.e. https://github.com/lsst-sqre/safir/blob/8930749f687c8f8e135d2926d00722478ba7dd1c/setup.cfg#L51

In this case, `requirements.txt` should also be dropped because all the relevant development packages are specified in the packaging config; `requirements.txt` is intended for apps rather than libraries.